Preview of the verification request

tortila opened the pull request:
**Scope of the changes:**
- the last step in the wizard (PREVIEW step) renders all cleaned data from the previous steps.
- new template for the PREVIEW step


**Out of scope** for now (will be delivered in the upcoming PRs):
- printing of the summary to a file
- "Edit" buttons for each section with hyperlinks to the corresponding wizard step

**Implementation details:**
- new `PreviewForm` that has no fields or logic and only serves as a placeholder in order to render an additional template at the end of the wizard steps,
- cleaned data from all previous steps is injected to the template context as unbound forms,
- the preview template renders each form in a separate section on the page.

I added comments in the code to describe all the weird bits of this implementation and I hope this will make it easier to extend in the future if need be.

tortila commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Show preview of the verification request before submitting it](https://trello.com/c/zHsPVDcN/74-show-preview-of-the-verification-request-before-submitting-it)

mrchrisadams commented on apps/accounts/templates/provider_registration/preview.html:
```diff
@@ -0,0 +1,77 @@
+            </section>
+            <section class="prose my-10">
+                <h3>Services provided by your organisation</h3>
+                {% include "provider_registration/partials/_preview.html" with form=preview_forms.2 %}
```
When I run this locally, I saw all the services offered, regardless of what I added. Can you please check if there is a missing conditional to only show the services a provider is actively offering?

tortila replied:
Good catch! I reverted the change (https://github.com/thegreenwebfoundation/admin-portal/pull/385/commits/6c226a19f3cf889218456bf658ac75806c03b97a) so that now we display the correct value (only selected services), but as slugs instead of human-readable names. I documented that in the linked Trello card so that we can tackle how to render the values separately from this PR.

mrchrisadams reviewed (commented):
Hi Oliwia, this looks good, thanks for creating it!

I have two short queries to resolve, then I'm happy to merge it in.

tortila commented:
@mrchrisadams based on our discussion, I changed the template logic to display only selected services (https://github.com/thegreenwebfoundation/admin-portal/commit/6c226a19f3cf889218456bf658ac75806c03b97a) and added some basic developer docs for the verification request form (https://github.com/thegreenwebfoundation/admin-portal/pull/385/commits/61096ab5f94ab6111e496e0fa8c9bfc55db29330). Please feel free to merge this PR if you're happy with the changes

mrchrisadams commented:
Hi @tortila - thank you for the extra bits, I'm happy to merge this in now.

Have a nice break!
